Clean up OSS guidelines (#454)
zalando/ost: _docs/index.md

  image: laptop.jpg
---

> #### Note: this page used to host Open Source guidelines for Zalandos. In 2022, the page has been moved to an internal documentation site, easily discoverable in our developer platform (Sunrise) and other well-known channels using "opensource" as keyword.

Author: The key changes are in this file.

Reviewer: Minor edits to make the content clearer both to Zalando employees and visitors from the web, plus removing some subjective wording:

```suggestion
#### Note: this page used to host Open Source guidelines for Zalando employees. In 2022, the page has been moved to an internal documentation site, discoverable in Zalando developer portal (Sunrise) and other internal channels using the keyword "opensource".
```

> Since moving this documentation to an internal documentation site, we extended the guidelines and approach as follows:

Reviewer: ```suggestion
After moving these guidelines to an internal site, we extended them as follows:
```

> 1. Replaced Google Forms with Github issues for tracking requests for releasing new Open Source projects and allow-listing of Licenses or CLAs.
> 2. Extended allow-listed CLAs with approval dates and additional notes from our legal team on considerations applying to our Zalando Group corporate structure.
> 3. Added a list of *Formerly Open Source Projects*, such as Akka, Redis, Terraform with recommended Open Source alternatives along with legal considerations for the licenses these projects migrated to.
> 4. Split release guidelines into two separate guides: *guide for maintainers* who would like to release a new project and a *guide for leads* who are asked for approval of open-sourcing a project.
> 5. Moved away from strict incubation and graduation processes to a more flexible approach, where projects are open-sourced on an ongoing basis and archived after a longer period of time of inactivity. We maintain two github organizations for historical reasons only.

Reviewer: ```suggestion
1. Replaced Google Forms with GitHub Issues to track requests for releasing new Open Source projects and allow-listing end-user licenses and CLAs (Contributor License Agreements).
2. Extended the list of allowed CLAs with approval dates and notes from the legal team.
3. Added a list of *Formerly Open Source Projects*, such as Akka, Redis, Terraform, together with legal considerations for the licenses these projects migrated to and with recommended Open Source alternatives.
4. Split release guidelines into two separate guides: *Guide for maintainers* who would like to release a new project and a *Guide for leads* who are asked for approval of open-sourcing a project.
5. Moved away from a strict incubation and graduation processes to a more flexible approach. Projects are now open-sourced continuously or archived after a long period of inactivity.
```

I am not sure if we need this sentence:
> Zalando maintains two GitHub organizations for historical reasons only.

The context for it is missing, namely: I don't remember what are these two orgs? I'd either elaborate on it in a new point (6.) or drop entirely to avoid confusion.

Author: Point on two orgs will be dropped.

> ### Archived OSS guidelines
> To view the archived content, check the [source code history](https://github.com/zalando/zalando.github.io/tree/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs) in github. The material includes:

Reviewer: ```suggestion
### Archived OSS guidelines
To view the archived content, check the [source code history](https://github.com/zalando/zalando.github.io/tree/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs) on GitHub. The material includes:
```

> - [Anti-Harassment Policy](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/resources/harassment-policy.md)
> - [Releasing a new project](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/releasing/index.md)
> - [Release sign-off guide for leads](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/releasing/lead-sign-off-guide.md)
> - [OSS Licensing guide](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/resources/licensing.md)
> - [Contributing upstream](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/using/contributing.md)
> - [Employee copyright](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/using/employee-copyright.md)

Reviewer: More consistent capitalization:

```suggestion
- [Anti-harassment policy](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/resources/harassment-policy.md)
- [Releasing a new project](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/releasing/index.md)
- [Release sign-off guide for leads](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/releasing/lead-sign-off-guide.md)
- [OSS licensing guide](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/resources/licensing.md)
- [Contributing upstream](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/using/contributing.md)
- [Employee copyright](https://github.com/zalando/zalando.github.io/blob/ca22fd9dfa96f6a25a6a5ea8b94c9ad815461cb4/_docs/using/employee-copyright.md)
```
